support-models/src/main/scala/com/gu/support/workers/Products.scala

```diff
@@ -31,11 +31,11 @@ case class Contribution(
 }
 
 case class SupporterPlus(
-    amount: BigDecimal,
+    preDiscountAmount: BigDecimal,
```

Comment:
renamed to make it clearer what it represents on a discounted sub.

Reply:
I don't think this is accurate though, currently (until we have a checkout which understands S+ discounts) this amount is the actual amount that the user will pay. I would revert this change but keep the others.

Reply:
the checkout itself doesn't understand S+ discounts but there will be lots of discounted S+ going in via the 3 tier checkout
top tier are going to be discounted S+ subscriptions with a separate discounted GW sub.
So the S+ will all have amount=10 but the actual amount we would charge the reader would be less than 10 pounds.  Ones that are >10 would still be correct.
I'm not sure how that relates to your comment though - perhaps I should call it `undiscountedAmount`?